src/wp-includes/classicpress/class-wp-block-type-pf.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Blocks API: WP_Block_Type class Polyfill
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 5.0.0
+ */
+
+/**
+ * Core class representing a block type.
+ *
+ * @since 5.0.0
+ *
+ * @see register_block_type()
+ */
+
+class WP_Block_Type {
+	public function __set( $name, $value ) {
```

Comment:
Given that none f the passed parameters are utilised should we maybe use `...$args` for each function?